digitalocean/README.md

```diff
@@ -0,0 +1,24 @@
+# DigitalOcean
+
+This small setup can be used to provision Flatcar nodes on [DigitalOcean](https://www.digitalocean.com/).
+
+Edit the [Container Linux Config](https://kinvolk.io/docs/flatcar-container-linux/latest/container-linux-config-transpiler/configuration/) `machine-mynode.yaml.tmpl` file if you like, then create the following `terraform.tfvars` with a machine `mynode`, corresponding to the Container Linux Config file name. If you add more machines, create new files for them.
+
+```
+cluster_name = "mycluster"
+machines     = ["mynode"]
+datacenter   = "nyc3"
+ssh_keys     = ["ssh-rsa AA... me@mail.net"]
+```
+
+Now run Terraform (version 13) as follows:
+
+```
+export DIGITALOCEAN_TOKEN=...
+terraform init
+terraform apply
+```
+
+Log in via `ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null core@IPADDRESS`.
```

Comment:
For non-ephemeral instances, this seems like a bad practice.

Reply:
For immutable infra where the whole instance is replaced it somehow makes sense, but I can turn this into a suggesstion.